android/shopify_buy_plugin/src/main/java/com/shopify/unity/buy/androidpay/AndroidPayCheckoutSession.java

```diff
@@ -59,19 +58,14 @@ public void canCheckoutWithAndroidPay(String cardBrandsString) {
         }
     }
 
-    public boolean checkoutWithAndroidPay(
+    public void checkoutWithAndroidPay(
             String merchantName,
             String publicKey,
             String pricingLineItemsString,
             String currencyCode,
             String countryCode,
             boolean requiresShipping
     ) {
-        if (!PayHelper.isAndroidPayEnabledInManifest(rootActivity)) {
```

Comment:
I suspect this check is not needed, since we explicitly add AP support to our manifest, right? Or is this something the client has to worry about?

Reply:
You're right - if we add it ourselves this check is redundant 🔥